storage/tools/queue_leaves/main.go

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// The queue_leaves binary queues a number of leaves for a log from a given
+// start point with predictable hashes.  If anything fails it panics, leaving
+// storage untouched
```

Comment:
I think we can probably remove these old binaries now, outside this PR of course.